### ISSUE TITLE
About suggestion for training and validation split of BlendedMVG

### ISSUE THREAD

**Issue by MiZhenxing** [YoYo000/BlendedMVS#18]
Hi, YaoYao,

Thank you very much for you work. I am planning to train my network with BlendedMVG (BlendedMVS, BlendedMVS+, BlendedMVS++). I would like to split the whole dataset into a training set and a validation set. Do you have any suggestion on which scenes in training and which scenes in validation and also how many scenes should in both splits? I have seen in your paper that you split the BlendedMVS. I would like tto know what split did you use when you training on BlendedMVG.

I also have another question. I see in the dataset that you also provided masked images. Do you train with masked or unmasked images?

Thank you very much!

**Comment by YoYo000** [comment]
Hi @MiZhenxing 

Previously I randomly select several more scenes for validation but forgot to upload the list. The list should be `BlendedMVG_validation.txt` = `BlendedMVG.txt` - `BlendedMVG_training.txt`, and it should be a superset of the current `validation_list.txt`.

I was using the masked image in BlendedMVS for my training. For the new BlendedMVS+/++, we managed to generate a complete image in a more natural way. You could try both and I think the difference is small.

**Comment by MiZhenxing** [comment]
Thank you very much. It is really helpful!
